src/cljs/rems/application.cljs

```diff
@@ -598,7 +598,8 @@
       :title [:span#application-state
               (str
                (text :t.applications/state)
-               (when state (str ": " (localize-state state))))]
+               (when state (str ": " (localize-state state)))
+               " — " (text :t.applications/last-modified) ": " (localize-time last-modified))]
```

Comment:
Why not include the whole text in the translation?
`(text-format :t.applications/last-modified (localize-time last-modified))`